frontend/src/components/TopNavigation.tsx

```diff
@@ -57,7 +57,7 @@ const TopNavigation: FunctionComponent<Props> = ({ hideControls }) => {
   )
 
   return (
-    <header className="py-2 bg-navy-800 h-nav sticky top-0">
+    <header className="py-2 bg-navy-800 h-nav sticky top-0 z-20">
```

Comment:
Unrelated UI bug where some elements with `position: absolute` would display on top of the sticky nav because they appear later in the HTML.